### ISSUE TITLE
Using Vite for dynamic module federation with React Hooks

### ISSUE THREAD

**Issue by yuvalbne** [iconag-bbasmer/vite-module-federation-test#1]
Hi,
Did you solve the issue you described in StackOverflow?
I am facing the same issue exactly.
https://stackoverflow.com/questions/76122146/using-vite-for-dynamic-module-federation-with-react-hooks

**Comment by iconag-bbasmer** [comment]
No, I couldn't solve it but I found out that the module federation plugin for Vite still seems to have a problem in this regard. So I decided to move away from using Vite. In the meantime I switched to use [Nx](https://nx.dev/) as a platform to manage my code in a monorepo and they have a working solution for module federation built-in. It uses Webpack under the hood I guess and it works without problem. It took me a while to understand and setup though.
